snippets/ztp_SriovOperatorConfig.yaml

```diff
@@ -20,6 +21,8 @@ spec:
   #          openshift.io/<resource_name>:  "1"
   #        requests:
   #          openshift.io/<resource_name>:  "1"
-  enableInjector: true
-  enableOperatorWebhook: true
+  enableInjector: false
+  enableOperatorWebhook: false
+  # Disable drain is needed for Single Node Openshift
```

Comment:
+1 to @maxwelldb's comments. Even though this is a comment in a YAML file, it is still customer-facing. 

/remove-label merge-review-needed
/remove-label merge-review-in-progress